lib/init.c

```diff
@@ -1081,6 +1085,11 @@ static int read_cfgfile(struct rpminspect *ri, const char *filename)
                             free(ri->profiledir);
                             ri->profiledir = strdup(t);
                         }
+                    } else if (block == BLOCK_COMMON) {
```

Comment:
It looks like BLOCK_COMMON was copied over from the above if statement and needs to be changed to BLOCK_ENVIRONMENT per line 691

Reply:
Fixed in followup commit, thanks!